terra_opp/tests/test_viewpoints.py

```diff
@@ -46,17 +46,14 @@ def setUp(self):
             os.path.join(os.path.dirname(__file__), 'placeholder.jpg'),
             'rb',
         )
-        date = timezone.datetime(2018, 1, 1, tzinfo=timezone.utc)
         self.data_create = {
             "label": "Basic viewpoint created",
             "point": self.feature.geom.json,
         }
         self.data_create_with_picture = {
             "label": "Viewpoint created with picture",
             "point": self.feature.geom.json,
-            # Cannot have nested json when working on files
-            "picture.date": date,
-            "picture.file": self.fp,
+            "picture_ids": [self.viewpoint_with_accepted_picture.pictures.first().pk],
```

Comment:
Why only the first picture ? Can we use the list of all id here, even if the viewpoint has only one.